packages/tasklist-client-node-js/src/__test__/restclient.spec.ts

```diff
@@ -44,130 +41,99 @@ describe('TasklistRESTClient', () => {
 
     afterAll(() => zbc.close())
 
-    it('can request all tasks', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getAllTasks()
-        console.log('tasks', tasks)
-        expect(tasks.length).toBeGreaterThan(0)
-    })
-
-    it('can request a task with parameters', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+    describe('Read operations', () => {
+        it('can request all tasks', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getAllTasks()
+            expect(tasks.length).toBeGreaterThan(0)
         })
-        expect(tasks.length).toBeGreaterThan(0)
-    })
 
-    it('can request tasks with fields', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+        it('can request a task with parameters', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({
+                state: TaskStateREST.CREATED,
+            })
+            expect(tasks.length).toBeGreaterThan(0)
         })
-        expect(tasks.length).toBeGreaterThan(0)
-        expect(tasks[0].name).not.toBeDefined()
-        expect(tasks[0].processName).toBeTruthy()
-    })
 
-    it('can request tasks with all fields', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({
-            state: TaskState.CREATED,
+        it('gets all fields for a task', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({
+                state: TaskStateREST.CREATED,
+            })
+            expect(tasks.length).toBeGreaterThan(0)
+            expect(tasks[0].name).toBeTruthy()
+            expect(tasks[0].processName).toBeTruthy()
```

Comment:
This test replaced _two_ tests on the left.

The GraphQL endpoint, on which these tests were originally based, supports specifying fields to return in the query. I understood the original two tests to confirm that the partial querying functionality worked. 

I replaced them with one test which confirms that fields are populated.